youtube_dl/extractor/wsj.py

```diff
@@ -17,6 +17,7 @@ class WSJIE(InfoExtractor):
         )
         (?P<id>[a-zA-Z0-9-]+)'''
     IE_DESC = 'Wall Street Journal'
+    TEMPLATE_URL = 'https://wsj.com/video/./%s'
```

Comment:
What's the point extracting it here? Also use http://video-api.wsj.com/api-video/player/iframe.html?guid=... instead or add a `wsj:` shortcut.